Assets/AWSIM/Scripts/Environments/V2I/V2I.cs

```diff
@@ -0,0 +1,69 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using GeometryUtility = AWSIM.Lanelet.GeometryUtility;
+
+namespace AWSIM
+{
+    public class V2I : MonoBehaviour
+    {
+        [SerializeField,
+        Tooltip("traffic_signals publication frequency - number of publications per second"),
+        Range(1.0f, 100.0f)]
+        int outputHz = 10;
+
+        private Vehicle egoVehicle;
+
+        private TrafficLight[] allTrafficLights;
+
+        public double egoDistanceToTrafficSignals = 150.0;
+        
+        public class OutputData
+        {
+            public TrafficLight[] trafficLights;
+        }
+
+        public delegate void OnOutputDataDelegate(OutputData outputData);
+        public OnOutputDataDelegate OnOutputData;
+
+        OutputData outputData = new OutputData();
+        float timer = 0;
+
+        void Start()
+        {
+            egoVehicle = GameObject.FindObjectOfType<Vehicle>();
```

Comment:
Could you have transform as a member of public and refer to it manually in the inspector?